presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5959,6 +5959,38 @@ public void testTry()
     }
 
     @Test
+    public void testLambdaFunctionWithTry()
+    {
+        assertQuery("SELECT TRY(apply(5, x -> x + 1) / 0)", "SELECT NULL");
+        assertQuery("SELECT TRY(apply(5 + RANDOM(1), x -> x + 1) / 0)", "SELECT NULL");
+        assertQueryFails("SELECT apply(5 + RANDOM(1), x -> x + TRY(1 / 0))", "Try expression inside lambda expression is not support yet");
```

Comment:
I think part of the motivation of this change is to make it possible to have try expression inside lambda expression?  Can we support it ? :)